src/ui/PopUpPosition.js

```diff
@@ -46,6 +46,20 @@ export function atAnchorBottomCenter(anchorRect: ?Rect, bodyRect: ?Rect): Rect {
   return rect;
 }
 
+export function atAnchorBottomLeft(anchorRect: ?Rect, bodyRect: ?Rect): Rect {
```

Comment:
Is this the same as `atAnchorBottom` currently? https://github.com/chanzuckerberg/czi-prosemirror/blob/master/src/ui/PopUpPosition.js#L9
(though it will be changing soon in https://github.com/chanzuckerberg/czi-prosemirror/pull/9)
I'm a little confused by the difference between `atAnchorBottom`, `atAnchorBottomCenter` and now `atAnchorBottomLeft`

Reply:
Looking at that more closely, looks like what I'm trying to do here is just a simpler version of `atAnchorBottom`. I think it's not worth adding this if I can achieve the same thing with `atAnchorBottom`. Going to shut down this PR

Reply:
I did end up modifying `atAnchorBottom` in https://github.com/chanzuckerberg/czi-prosemirror/pull/9 so it may no longer be what you want, since it's now offset by half the anchor width.

Reply:
Haha ok, let's check when everything is on master